src/content/pages/pyladies.md

```diff
@@ -19,39 +19,60 @@ attendees an exclusive opportunity to connect, share, and grow together.
 Join us for a special lunch event aimed at fostering community and empowerment
 in tech. Enjoy meaningful conversations and networking opportunities.
 
-WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
+* WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
+* WHERE? Prague Congress Centre (PCC), VIP lunch area.
 
-WHERE? Prague Congress Centre (PCC), VIP lunch area.
+<div class="text-center mb-8">
+<ButtonLink href="https://forms.gle/ts9ryAQj8ieh36NB7">Register for the PyLadies Lunch</ButtonLink>
+</div>
+
+
+## PyLadies Open Space
+
+During this one-hour PyLadies Open Space session, we invite Pyladies
+organizers, members, newcomers, and allies to join a thoughtful and engaging
+dialogue.
+
+The aim of this session is to discuss the current status of our individual
+PyLadies communities, identify shared challenges, and explore the potential
+benefits of increased collaboration between different PyLadies groups.
+
+Through open and inclusive conversations, we hope to build stronger
+connections, foster mutual growth, and create opportunities for collective
+impact within our diverse PyLadies network.
+
+* WHEN? Thursday, 17th July 2025, 12:30 to 14:00.
+* WHERE? Prague Congress Centre (PCC), VIP lunch area.
 
-## More Events Coming Soon!
 
-<!--
 ## #IAmRemarkable
-Empower yourself in this workshop designed to help you celebrate your achievements and improve your self-promotion skills.
 
-WHEN?
-Thursday, 17th July 2024, 15:30 to 17:00.
+Empower yourself in this workshop designed to help you celebrate your
+achievements and improve your self-promotion skills.
 
-WHERE?
-Prague Congress Centre (PCC), Open Space 2 (Room 224)
+* WHEN?  Thursday, 17th July 2025,
+* WHERE? Prague Congress Centre (PCC), Open Space 2 (Room 224)
+* HOST: Daria Linhart Grudzien
+
+<div class="text-center mb-8">
+<ButtonLink href="https://forms.gle/6UEbDiHvTZGVkGWM9">Register for IAmRemarkable</ButtonLink>
+</div>
 
 ![](./images/iamremarkable.png)
 
 
-## Meet & Greet with PyLadies
-Take advantage of this unique networking opportunity to meet and connect with experienced members of PyLadies. Gain valuable insights, advice, and inspiration for your Python and life journeys.
+## Stretching Session
```

Comment:
If this is independent, then it needs a time and room, or in case it's inside IAmRemarkable  we can clarify it.

Reply:
adding a placeholder then.